sqlglot/parser.py

```diff
@@ -1219,7 +1225,23 @@ def _parse_sort(self, token_type, exp_class):
     def _parse_ordered(self):
         this = self._parse_conjunction()
         self._match(TokenType.ASC)
-        return self.expression(exp.Ordered, this=this, desc=self._match(TokenType.DESC))
+        is_desc = self._match(TokenType.DESC)
+        is_nulls_first = self._match(TokenType.NULLS_FIRST)
+        is_nulls_last = self._match(TokenType.NULLS_LAST)
+        desc = is_desc or False
```

Comment:
let's remove or False's, i'll probably make a PR that removes keys that are None

Reply:
we don't need to make desc required right?

Reply:
I see it as an improvement if it is. If there is no benefit from having 3 states vs. 2 then having 2 seems better.